opm/input/eclipse/Schedule/Well/Well.hpp

```diff
@@ -369,6 +369,7 @@ class Well {
 
         void init_history(const DeckRecord& record);
         void init_vfp(const std::optional<VFPProdTable::ALQ_TYPE>& alq_type, const int vfp_table_nr, const UnitSystem& unit_system, const DeckRecord& record);
+        void updateALQDimension(const VFPProdTable& vfptable, const UnitSystem& usys);
```

Comment:
This function (`Well::WellProductionProperties::updateALQWDimension()`) appears to be unimplemented, at least in the current PR.  Is that intentional?

Reply:
Eh, no - just forgot to delete from the first iteration, sorry about that. Will fix!